pcap_ex.c

```diff
@@ -27,15 +27,15 @@ int
 pcap_ex_immediate(pcap_t *pcap)
 {
 #ifdef _WIN32
-    return pcap_setmintocopy(pcap, 1);
+	return pcap_setmintocopy(pcap, 1);
```

Comment:
I guess it was automatic formatting from Sublime Text. Frankly, looks like pcap_ex is redundant and can be replaced by extending pyx-code. But it's out of scope of the task.